bin/publish.sh

```diff
@@ -1,41 +0,0 @@
-#!/bin/bash
-
-# Copied from: http://stackoverflow.com/a/246128
-__FILE_DIR__="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-# Copies FSharp.sublime-package to Packages/Installed Packages.
-publish_package () {
-    cp -f dist/FSharp.sublime-package "$data_path/Installed Packages"
-}
-
-# Copies FSharp's tests to Packages/FSharp_Tests.
-publish_tests () {
-    mkdir -p "$data_path/Packages/FSharp_Tests"
-    pushd "$__FILE_DIR__/../FSharp_Tests"
-        # Copy support files.
-        cp -f test_runner.py "$data_path/Packages/FSharp_Tests"
-        cp -f FSharpTests.sublime-commands "$data_path/Packages/FSharp_Tests"
```

Comment:
I did not implement installation of tests, because this file was missing, so I wasn't sure what to do. :)